cardano-testnet/test/cardano-testnet-test/files/golden/tx.failed.response.yaml.golden

```diff
@@ -0,0 +1,21 @@
+contents:
+  contents:
+    contents:
+      era: ShelleyBasedEraBabbage
+      error:
+      - contents:
+          contents: AlonzoInBabbageUtxoPredFailure (ValueNotConservedUTxO (MaryValue
+            (Coin 0) (MultiAsset (fromList []))) (MaryValue (Coin 300000000000) (MultiAsset
+            (fromList []))))
+          tag: UtxoFailure
+        tag: UtxowFailure
+      - contents:
+          contents: AlonzoInBabbageUtxoPredFailure (BadInputsUTxO (fromList [TxIn
+            (TxId {unTxId = SafeHash "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"})
+            (TxIx 0)]))
+          tag: UtxoFailure
+        tag: UtxowFailure
+      kind: ShelleyTxValidationError
+    tag: TxValidationErrorInCardanoMode
+  tag: TxCmdTxSubmitValidationError
+tag: TxSubmitFail
```

Comment:
This is the response that will be returned.  The golden file is yaml, but the response will be the JSON equivalent.

Reply:
What's the benefit of yaml vs pretty JSON?